cmd/check_test.go

```diff
@@ -43,30 +44,13 @@ var _ = Describe("cmd package check command", func() {
 			os.Unsetenv("PFLT_PYXIS_ENV")
 			os.Unsetenv("PFLT_PYXIS_HOST")
 		})
-		Context("Regular Connect URL", func() {
```

Comment:
I feel like these 2 tests should still be valid.

Reply:
Yes, I agree.  I can't remember why they were removed.  I'll add them back.

Reply:
Added the tests back.